File: src/MovieStore/MainMenu.cs
using Spectre.Console;
using System.Data;

namespace MovieStore
{
    internal static class MainMenu
    {
        const string SearchForTitle = "Search for title...";
        const string Quit = "Quit";
        internal static void Run(IDbConnection connection)
        {
            while (true)
            {
                AnsiConsole.Clear();
                AnsiConsole.MarkupLine("[olive]Main Menu:[/]");
                AnsiConsole.MarkupLine($"Titles: {Titles.Count(connection)}");
                var prompt = new SelectionPrompt<string>() { Title=string.Empty };
                prompt.AddChoice(SearchForTitle);
                prompt.AddChoice(Quit);
                switch(AnsiConsole.Prompt(prompt))
                {
                    case SearchForTitle:
                        TitleSearch.Run(connection); 
                        break;
                    case Quit:
                        return;
                }
            }
        }
    }
}

File: src/MovieStore/MainMenu.vb
Imports Microsoft.Data.SqlClient
Imports Spectre.Console

Friend Module MainMenu
    Const QuitChoice = "Quit"
    Const MainMenuPrompt = "[olive]Main Menu:[/]"
    Friend Sub Run(connection As SqlConnection)
        Do
            AnsiConsole.Clear()
            Dim prompt As New SelectionPrompt(Of String) With {.Title = "[olive]Main Menu:[/]"}
            prompt.AddChoice(QuitChoice)
            Select Case AnsiConsole.Prompt(prompt)
                Case QuitChoice
                    Exit Do
            End Select
        Loop
    End Sub
End Module
